repository.json

```diff
@@ -793,6 +795,26 @@
 				}
 			]
 		},
+		{
+			"details": "https://github.com/PythonVoiceCodingPlugin/LSP-pyvoice",
+			"name": "LSP-pyvoice",
+			"author": "mpourmpoulis",
+			"labels": [
+				"voice coding",
+				"accessibility"
+			],
+			"platforms": [
+				"windows",
+				"linux",
+				"osx"
+			],
```

Comment:
This can be removed :)

Reply:
removed it, 
also would you like me to squash my commits?